buildSrc/src/main/java/org/opensearch/gradle/testclusters/OpenSearchNode.java

```diff
@@ -1182,6 +1182,7 @@ private void createConfiguration() {
         baseConfig.put("indices.breaker.total.use_real_memory", "false");
         // Don't wait for state, just start up quickly. This will also allow new and old nodes in the BWC case to become the master
         baseConfig.put("discovery.initial_state_timeout", "0s");
+        baseConfig.put("fips.approved", "true");
```

Comment:
When we proposed a similar flag previously we were encouraged not to use boolean flags to control behaviour like this https://github.com/opensearch-project/security/issues/3420#issuecomment-2077610480

Reply:
I am of the opinion that this is fine, in this case, I am not sure what greater configuration would make sense. Unless you can provide different approvers or something, there is no reason for this not to be a boolean.

Reply:
I might have missed the part of the thread where that issue came up - I'd rather see `crypto.standard=FIPS-140-2` & `cypto.standard=any-supported` to help understand what state the server is in when viewing a configuration file.  I am definately open to other ideas that clarify what state the cluster is running it, as I think it could be useful to support FIPS-140-3 seperately or other standards/restrictions as they are addopted.

This isn't a blocker - it could be circled back on when we need to support a 3rd mode, the question is if that is a potential near term issue vs later on (or never).